retroarch/riscv64.patch

```diff
@@ -0,0 +1,38 @@
+--- PKGBUILD	2023-09-10 00:55:26.455719410 -0700
++++ PKGBUILD-riscv	2023-09-10 00:52:21.902014518 -0700
+@@ -6,13 +6,12 @@
+ pkgver=1.15.0
+ pkgrel=3
+ pkgdesc='Reference frontend for the libretro API'
+-arch=(x86_64)
++arch=('x86_64' 'riscv64')
```

Comment:
We don't need to change here for a minimal patch, our build script will do this automatically. :)